scripts/covid19_india/medical_tests_in_data/COVID19_tests_india.tmcf

```diff
@@ -0,0 +1,7 @@
+Node: E:COVID19_tests_india->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19
+measurementMethod: dcs:Covid19IndiaProject
+observationAbout: C:COVID19_tests_india->isoCode
+observationDate: C:COVID19_tests_india->Date
+value: C:COVID19_tests_india->CumulativeCount_MedicalTest_COVID_19
```

Comment:
Since we are planning to resolve using the "isoCode", I think the template MCF should be:

Node: E:COVID19_tests_india->E0
typeOf: dcs:StatVarObservation
variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19
measurementMethod: dcs:Covid19IndiaProject
observationAbout: E:COVID19_tests_india->E1
observationDate: C:COVID19_tests_india->Date
value: C:COVID19_tests_india->CumulativeCount_MedicalTest_COVID_19

Node: E:COVID19_tests_india->E1
typeOf: schema:Country
isoCode: C:COVID19_tests_india->isoCode